app/src/test/java/com/teamagam/gimelgimel/app/model/ViewsModels/MessageContentUserLocationUnitTest.java

```diff
@@ -0,0 +1,48 @@
+package com.teamagam.gimelgimel.app.model.ViewsModels;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.teamagam.gimelgimel.app.model.entities.LocationSample;
+import com.teamagam.gimelgimel.app.view.viewer.data.geometries.PointGeometry;
+
+import org.junit.Test;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Yoni on 4/18/2016.
+ * Tester for {@link MessageContentUserLocation}
```

Comment:
I think its kinda obvious and that test classes shouldn't be documented unless something there's something interesting to pay attention to